src/object.c

```diff
@@ -1812,47 +1837,247 @@ void func_80023D08(Object *obj, u16 param2) {
 // start of objanim.c or something like that?
 #pragma GLOBAL_ASM("asm/nonmatchings/object/func_80023D30.s")
 
+#if 1
 #pragma GLOBAL_ASM("asm/nonmatchings/object/func_800240BC.s")
+#else
+s32 func_800240BC(Object* object, f32 progress) {
```

Comment:
No reason to keep this if-ed out, there's a match from you from 4 months ago: https://decomp.me/scratch/RUVe8 ;)